File: html-converter-dotnet-core-vb/JavaScriptCssConversion.vb
Imports ceTe.DynamicPDF.HtmlConverter

Class JavaScriptCssConversion

    Private Shared samplePageWithCss As String = "<!DOCTYPE html><html><head ><style>" _
            & "body {background-color: lightblue;}" _
            & "h1 {color: white; text-align: center;} p " _
            & "{font-family: verdana;font-size: 20px;}</style>" _
            & "</head><body><h1>My First CSS Example</h1>" _
            & "<p>This is a paragraph.</p></body></html>"

    Private Shared samplePageWithJavaScript As String = "<!DOCTYPE html><html><body>" _
            & "<h2>My First Web Page</h2>" _
            & "<p>My First Paragraph.</p><p id=""demo""></p>" _
            & "<script>document.getElementById(""demo"")" _
            & ".innerHTML = 5 + 6;</script></body></html>"

    Public Shared Sub Run()
        Converter.Convert(samplePageWithCss, Util.GetPath("Output/cssExample.pdf"))
        Converter.Convert(samplePageWithJavaScript, Util.GetPath("Output/jscriptExample.pdf"))
    End Sub

End Class

File: html-converter-dotnet-core-cs/JavaScriptCssConversion.cs
using ceTe.DynamicPDF.HtmlConverter;

namespace html_converter_dotnet_core_cs
{
    class JavaScriptCssConversion
    {

        static string samplePageWithCss = "<!DOCTYPE html><html><head ><style>" 
            + "body {background-color: lightblue;}"
            + "h1 {color: white; text-align: center;} p "
            + "{font-family: verdana;font - size: 20px;}</style>"
            + "</head><body><h1>My First CSS Example</h1>"
            + "<p>This is a paragraph.</p></body></html>";

        static string samplePageWithJavaScript = "<!DOCTYPE html><html><body>"
            + "<h2>My First Web Page</h2>"
            + "<p>My First Paragraph.</p><p id=\"demo\"></p>"
            + "<script>document.getElementById(\"demo\")"
            + ".innerHTML = 5 + 6;</script></body></html>";

        public static void Run()
        {
            Converter.Convert(samplePageWithCss, Util.GetPath("Output/cssExample.pdf"));
            Converter.Convert(samplePageWithJavaScript, Util.GetPath("Output/jscriptExample.pdf"));
        }
    }
}
